src/lineage/snps/reader.py

```diff
@@ -373,3 +376,66 @@ def read_generic_csv(file):
         )
 
         return df, "generic"
+
+    @staticmethod
+    def read_vcf(file):
+        """ Read and parse VCF file.
+
+        Notes
+        -----
+        This function uses the PyVCF python module to parse the genotypes from VCF files:
+        https://pyvcf.readthedocs.io/en/latest/index.html
+
+
+        Parameters
+        ----------
+        file : str
+            path to file
+
+        Returns
+        -------
+        pandas.DataFrame
+            genetic data normalized for use with `lineage`
+        str
+            name of data source
+        """
+        df = pd.DataFrame(columns=["rsid", "chrom", "pos", "genotype"])
+        df = df.astype(
+            {"rsid": object, "chrom": object, "pos": np.int64, "genotype": object}
+        )
+
+        vcf_reader = vcf.Reader(open(file, "r"))
+
+        # lineage does not yet support multi-sample vcf.
+        if len(vcf_reader.samples) > 1:
+            return None, "vcf"
```

Comment:
I struggled with the return here, it feels like there should be some notification the user that no file is being read, via `logging` or `warnings`?

Reply:
Good call. I think for now a simple `print` statement should be sufficient and consistent with status printed throughout the library.